src/theory/sets/theory_sets_private.cpp

```diff
@@ -535,11 +527,13 @@ void TheorySetsPrivate::fullEffortCheck(){
 
     std::vector< Node > lemmas;
     Trace("sets-eqc") << "Equality Engine:" << std::endl;
+    std::map<TypeNode, unsigned> eqcTypeCount;
```

Comment:
Hmm, I am wondering whether we should guard this to be enabled only in tracing builds. It seems like it is only used for debugging. Probably doesn't have a huge impact on performance though.

Reply:
I thought about this, but yeah, this shouldn't be a big deal.  Makes the code really ugly otherwise.